<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ftp</class>
 <widget class="QWidget" name="Ftp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ftp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>FTP Address:</string>
       </property>
       <property name="buddy">
        <cstring>FtpAddress</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="FtpAddress">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>10.200.19.16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>192.168.0.107</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="FtpPort">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>21</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAbort">
       <property name="text">
        <string>Abort</string>
       </property>
       <property name="icon">
        <iconset resource="../IDE.qrc">
         <normaloff>:/Menu/rc/Menu/stop.png</normaloff>:/Menu/rc/Menu/stop.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="icon">
        <iconset resource="../IDE.qrc">
         <normaloff>:/Menu/rc/Menu/refresh.png</normaloff>:/Menu/rc/Menu/refresh.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="icon">
        <iconset resource="../IDE.qrc">
         <normaloff>:/Menu/rc/Menu/notpass.png</normaloff>:/Menu/rc/Menu/notpass.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="AuthenticationGroup">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Authentication(anonymity)</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>UserName:</string>
          </property>
          <property name="buddy">
           <cstring>UserName</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UserName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Passwd"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>37</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>37</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Local:</string>
           </property>
           <property name="buddy">
            <cstring>UserName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LocalCatalog"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="Local_Back">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/left.png</normaloff>:/Menu/rc/Menu/left.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Local_Next">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/right.png</normaloff>:/Menu/rc/Menu/right.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Local_Upwards">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/cdparent.png</normaloff>:/Menu/rc/Menu/cdparent.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="LocalFileAdd">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="LocalFileDel">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/minus.png</normaloff>:/Menu/rc/Menu/minus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="BtnUpLoad">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../IDE.qrc">
           <normaloff>:/Menu/rc/Menu/next.png</normaloff>:/Menu/rc/Menu/next.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnDownLoad">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../IDE.qrc">
           <normaloff>:/Menu/rc/Menu/previous.png</normaloff>:/Menu/rc/Menu/previous.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="Remote">
           <property name="minimumSize">
            <size>
             <width>43</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>43</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Remote:</string>
           </property>
           <property name="buddy">
            <cstring>UserName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RemoteCatalog"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="Upwards">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/cdparent.png</normaloff>:/Menu/rc/Menu/cdparent.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="RemoteFileAdd">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoteFileDel">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/minus.png</normaloff>:/Menu/rc/Menu/minus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../IDE.qrc"/>
 </resources>
 <connections/>
</ui>
